spm_funcs.py

```diff
@@ -37,6 +37,7 @@ def spm_global(vol):
     g : float
         SPM global metric for `vol`
     """
+
```

Comment:
Typically look to delete or leave out blank lines unless they made a reasonably large difference to readability - otherwise they can make the code longer, less concise.